src/main/java/qna/domain/Question.java

```diff
@@ -14,11 +17,16 @@ public class Question extends BaseEntity {
     @Lob
     private String contents;
 
-    private Long writerId;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "writer_id", foreignKey = @ForeignKey(name = "fk_question_writer"))
+    private User writer;
```

Comment:
질문 생성시 작성자가 필수라면 optional = false 옵션을 추가해줄 수 있겠네요
```suggestion
    @ManyToOne(optional = false, fetch = FetchType.LAZY)
    @JoinColumn(name = "writer_id", foreignKey = @ForeignKey(name = "fk_question_writer"))
    private User writer;
```